Week2/Sum3test.java

```diff
@@ -0,0 +1,66 @@
+package sum3;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.testng.annotations.DataProvider;
+
+public class Sum3test {
+  @Test
+  public void f() {
```

Comment:
What do these test method names mean? If a test method is named after what it is testing, it is a lot easier to debug and diagnose failures

Reply:
sure. I will take care of it.